python/conda/libastra/meta.yaml

```diff
@@ -8,21 +8,22 @@ source:
 
 build:
   number: 0
-  script_env:
-    - CC # [linux]
-    - CXX # [linux]
+  string: cuda_{{ cudatoolkit }}
```

Comment:
This needs a `[linux]` tag since the `cudatoolkit` dependency is linux-only.

Reply:
Thanks. Fixed!